crates/common/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+//! Spin common modules
```

Comment:
Does this need a mandatory docs thingy, or are we not too concerned because these things are "logically internal", or does it inherit one from somewhere else?